bin/cliain/src/commands.rs

```diff
@@ -0,0 +1,228 @@
+        /// Nominator lower bound
+        #[clap(long)]
+        minimal_nominator_stake: u64,
+
+        /// Validator lower bound
+        #[clap(long)]
+        minimal_validator_stake: u64,
+
+        /// Maximum number of nominators
+        #[clap(long)]
+        max_nominators_count: Option<u32>,
+
+        /// Maximum number of validators
+        #[clap(long)]
+        max_validators_count: Option<u32>,
+    },
+
+    /// Transfer funds via balances pallet
+    Transfer {
+        /// Number of tokens to send,
+        #[clap(long)]
+        amount_in_tokens: u64,
+
+        /// SS58 id of target account
+        #[clap(long)]
+        to_account: String,
+    },
+
+    /// Send new runtime (requires sudo account)
+    UpdateRuntime {
+        #[clap(long)]
+        /// Path to WASM file with runtime
+        runtime: String,
+    },
+
+    /// Call staking validate call for a given controller
+    Validate {
+        /// Validator commission percentage
+        #[clap(long)]
+        commission_percentage: u8,
+    },
+
+    /// Update vesting for the calling account.
+    Vest,
+
+    /// Update vesting on behalf of the given account.
+    VestOther {
+        /// Account seed for which vesting should be performed.
+        #[clap(long)]
+        vesting_account: String,
+    },
+
+    /// Transfer funds via balances pallet
+    VestedTransfer {
+        /// Number of tokens to send.
+        #[clap(long)]
+        amount_in_tokens: u64,
+
+        /// Seed of the target account.
+        #[clap(long)]
+        to_account: String,
+
+        /// How much balance (in rappens, not in tokens) should be unlocked per block.
+        #[clap(long)]
+        per_block: Balance,
+
+        /// Block number when unlocking should start.
+        #[clap(long)]
+        starting_block: BlockNumber,
+    },
+
+    /// Print debug info of storage
+    DebugStorage,
+
+    /// Uploads new code without instantiating a contract from it
+    /// https://polkadot.js.org/docs/substrate/extrinsics/#uploadcodecode-bytes-storage_deposit_limit-optioncompactu128
+    ContractUploadCode {
+        /// Path to the .wasm artifact
+        #[clap(long, parse(from_os_str))]
+        wasm_path: PathBuf,
+        /// The maximum amount of balance that can be charged/reserved from the caller to pay for the storage consumed
+        #[clap(long)]
+        storage_deposit_limit: Option<u128>,
+    },
+
+    ///  Instantiates a contract from a previously deployed wasm binary.
+    /// API signature: https://polkadot.js.org/docs/substrate/extrinsics/#instantiatevalue-compactu128-gas_limit-compactu64-storage_deposit_limit-optioncompactu128-code_hash-h256-data-bytes-salt-bytes
+    ContractInstantiate {
+        /// balance to transfer from the call origin to the contract
+        #[clap(long, default_value = "0")]
+        balance: u128,
+        /// The gas limit enforced when executing the constructor
+        #[clap(long, default_value = "1000000000")]
```

Comment:
nit: can we use here `1_000_...` syntax here?

Reply:
fixed